garrysmod/gamemodes/terrortown/gamemode/corpse.lua

```diff
@@ -386,6 +386,11 @@ function CORPSE.Create(ply, attacker, dmginfo)
 
    rag:SetPos(ply:GetPos())
    rag:SetModel(ply:GetModel())
+   rag:SetSkin(ply:GetSkin())
+   bodyGroupTable = ply:GetBodyGroups()
+   for id in pairs(bodyGroupTable) do

```

Comment:
You're creating a `bodyGroupTable` global var here. Just make it `for id in pairs(ply:GetBodyGroups()) do`